src/mysql.erl

```diff
@@ -668,11 +670,12 @@ handle_call(backslash_escapes_enabled, _From, State = #state{status = S}) ->
     {reply, S band ?SERVER_STATUS_NO_BACKSLASH_ESCAPES == 0, State};
 handle_call(in_transaction, _From, State) ->
     {reply, State#state.status band ?SERVER_STATUS_IN_TRANS /= 0, State};
-handle_call(start_transaction, _From,
+handle_call(start_transaction, {From, _Tag},
             State = #state{socket = Socket, transaction_level = L,
                            status = Status})
   when Status band ?SERVER_STATUS_IN_TRANS == 0, L == 0;
        Status band ?SERVER_STATUS_IN_TRANS /= 0, L > 0 ->
+    erlang:link(From),
```

Comment:
How about nested transactions? We only want to create a link if it is the outermost transaction, right? (Or is it possible to have multiple links between two processes?)